worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/io/FastSchematicReaderV3.java

```diff
@@ -136,6 +139,24 @@ public Clipboard read(final UUID uuid, final Function<BlockVector3, Clipboard> c
                     this.dataVersion = this.dataInputStream.readInt();
                     this.dataFixer = ReaderUtil.getVersionedDataFixer(this.dataVersion, platform, platform.getDataVersion());
                 }
+                case "Metadata" -> {
+                    LinCompoundTag metadataCompoundTag =
+                            (LinCompoundTag) this.nbtInputStream.readTagPayload(NBTConstants.TYPE_COMPOUND, 2).toLinTag();
```

Comment:
Any specific reason using 2 as the initial depth over 0?

Reply:
I'll be honest, that is left-over from when I was familiarizing myself with the codebase.
Let me push a commit real quick to change that to `0` (tested and working).